<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteEditorForm</class>
 <widget class="QWidget" name="RouteEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Map</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="OSMWidget" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Locations</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="9">
       <widget class="QListWidget" name="listWidget_locations"/>
      </item>
      <item row="3" column="1" colspan="8">
       <widget class="QLineEdit" name="lineEdit_location_pos"/>
      </item>
      <item row="1" column="6">
       <widget class="QToolButton" name="toolButton_add_location">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/gui/icons/resources/add.png</normaloff>:/gui/icons/resources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>111</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="7">
       <widget class="QToolButton" name="toolButton_edit_location">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/gui/icons/resources/edit.png</normaloff>:/gui/icons/resources/edit.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QToolButton" name="toolButton_remove_location">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/gui/icons/resources/remove.png</normaloff>:/gui/icons/resources/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="8">
       <widget class="QLineEdit" name="lineEdit_location_name"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Loc:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Routes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="5">
       <widget class="QToolButton" name="toolButton_route_delete">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/gui/icons/resources/remove.png</normaloff>:/gui/icons/resources/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButton_add_route">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/gui/icons/resources/add.png</normaloff>:/gui/icons/resources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_route_name"/>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_route_visible">
        <property name="text">
         <string>Visible</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="toolButton_save_routes">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/gui/icons/resources/save.png</normaloff>:/gui/icons/resources/save.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="toolButton_route_edit">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/gui/icons/resources/edit.png</normaloff>:/gui/icons/resources/edit.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QListWidget" name="listWidget_route_list"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBox_polygon_closed">
        <property name="text">
         <string>Is closed</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkBox_route_is_editable">
        <property name="text">
         <string>Editable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OSMWidget</class>
   <extends>QWidget</extends>
   <header>OSMWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
